test/assert.rego

```diff
@@ -61,6 +61,42 @@ not_has(item, coll) if {
 	print("expected", type_name(item), _quote_str(item), "not in", type_name(coll), "got:", coll)
 }
 
+# METADATA
+# description: Assert key is in obj
+has_key(key, obj) if {
+	some keys, _ in obj
+	key == keys
```

Comment:
Ah, right. The reason why I didn't use it was because it doesn't appear to be supported in Jarl, but that isn't an issue here. Which do you prefer here?

Reply:
Didn't notice that you merged already. That was fast, thanks! :sunglasses:

Reply:
Oh, you're using Jarl? That's awesome! I'd love to hear more about that :)

Also I just pushed a minor change to main to better align the directory structure to that of the package https://github.com/anderseknert/rego-test-assertions/commit/7533e18d8c9fa07e09d2bc4aa3b0efe7cdf343b3

Feel free to ignore :)

Reply:
I haven't had the time to hack on Jarl recently :/ but adding the few built-ins that have been added to OPA recently shouldn't be too much work... might get around to it one day. And of course, feel free to submit fixes there if you find something missing.

Reply:
Yeah, I'm using Jarl for policy driven authorization in a Spring application. Unfortunately, I can't share much (and it's admittedly a bit hairy so far), but I created a minimal application for test purposes earlier today, which is similar to what I'm working on: https://gitlab.com/fleskesvor/auth-server

I'm not very familiar with Clojure (yet), but I'll see if there are things I can contribute to make it easier to use.

Also, I've made that same change in directory structure in the project where I use this library to make it pass linting with regal. :slightly_smiling_face: